source/security/enterprise-authentication.txt

```diff
@@ -0,0 +1,410 @@
+
+.. _java-rs-plain-auth-mechanism:
+
+LDAP (PLAIN)
+~~~~~~~~~~~~
+
+You can authenticate to a Lightweight Directory Access Protocol (LDAP)
+server by using your directory server username and password.
+
+.. tip::
+
+   The authentication mechanism is named ``PLAIN`` instead of ``LDAP`` since it
+   authenticates using the `PLAIN Simple Authentication and Security Layer
+   (SASL) defined in RFC-4616 <https://tools.ietf.org/html/rfc4616>`_.
+
+The following sections contain code examples that use the following placeholders:
+
+- ``ldap_username``: your LDAP username
+- ``ldap_password``: your LDAP user's password
+- ``hostname``: network address of your MongoDB deployment that your
+  client can access
+- ``port``: port number of your MongoDB deployment
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tabs to
+see the corresponding syntax.
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      .. literalinclude:: /includes/security/enterprise-authentication.java
+         :start-after: // start-ldap-connection-string
+         :end-before: // end-ldap-connection-string
+         :language: java
+
+   .. tab::
+      :tabid: MongoCredential
+
+      To specify the LDAP (PLAIN) authentication mechanism by using the
+      ``MongoCredential`` class, call the ``createPlainCredential()``
+      method, as shown in the following example:
+
+      .. literalinclude:: /includes/security/enterprise-authentication.java
+         :start-after: // start-ldap-mongocredential
+         :end-before: // end-ldap-mongocredential
+         :language: java
+
+.. _java-rs-mongodb-oidc:
+
+MONGODB-OIDC
+~~~~~~~~~~~~
+
+.. important::
+
+   The MONGODB-OIDC authentication mechanism requires {+mdb-server+} v7.0 or later running
+   on a Linux platform.
+
+The following sections describe how to use the MONGODB-OIDC authentication mechanism to
+authenticate to various platforms.
+
+For more information about the MONGODB-OIDC authentication mechanism, see
+:manual:`OpenID Connect Authentication </core/security-oidc/>` and
+:manual:`MongoDB Server Parameters </reference/parameters/#mongodb-parameter-param.oidcIdentityProviders>`
+in the MongoDB Server manual.
+
+.. _java-rs-mongodb-oidc-azure-imds:
+
+Azure IMDS
+++++++++++
+
+If your application runs on an Azure VM, or otherwise uses the
+`Azure Instance Metadata Service <https://learn.microsoft.com/en-us/azure/virtual-machines/instance-metadata-service>`__
+(IMDS), you can authenticate to MongoDB by using the {+driver-short+}'s built-in Azure
+support.
+
+You can specify Azure IMDS OIDC authentication either by 
+using a ``MongoCredential`` or as part of the connection string.
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tabs to
+see the corresponding syntax.
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: mongodb-azure-imds-connection-string
+    
```

Comment:
S: since there's also a db_username placeholder, I'd instruct the reader to replace that as well